interface/globals.php

```diff
@@ -465,4 +465,16 @@ function strterm($string,$length) {
   extract($_GET,EXTR_SKIP);
   extract($_POST,EXTR_SKIP);
 }
+//Create page validation array -- all the pahges that have to fire validatejs and their rules
+if (!empty($_SESSION['authUserID'])) {
+    $pages = sqlStatement("SELECT * " .
+        "FROM `list_options` WHERE list_id='PageValidation'");
+
+    for($iter=0; $row=sqlFetchArray($pages); $iter++) {
+        //remove global_ prefix from label
+        $_GLOBALS['validate_pages'][$row['option_id']]= array('page_name'=>$row['option_id'].".php", 'rules'=>$row['notes']);
+    }
+
+}
+
```

Comment:
Will this scale up? Just seems like this would get very large in size if every script used this?
Rather than build the global, couldn't each page that has validation support do this instead?